strongbox-data-service/src/test/java/org/janusgraph/graphdb/database/idassigner/placement/StaticPlacementStratrgy.java

```diff
@@ -0,0 +1,49 @@
+package org.janusgraph.graphdb.database.idassigner.placement;
+
+import java.util.Map;
+
+import org.janusgraph.core.JanusGraphVertex;
+import org.janusgraph.diskstorage.configuration.Configuration;
+import org.janusgraph.graphdb.configuration.PreInitializeConfigOptions;
+import org.janusgraph.graphdb.internal.InternalElement;
+import org.janusgraph.graphdb.internal.InternalVertex;
+
+@PreInitializeConfigOptions
+public class StaticPlacementStratrgy extends SimpleBulkPlacementStrategy
```

Comment:
There is a typo in the name of this class. It should be renamed to `StaticPlacementStrategy`.

Reply:
fixed